### PR TITLE
Updating with TextExtracts params for GP

### PR DESCRIPTION
**Pull request by ugochimobi**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3956,6 +3956,21 @@
...
+		'default' => false,
+		'gratispaideiawiki' => [
+			'.metadata', // This is a good thing to remove in any case
+			'span.coordinates',
+			'span.geo-multi-punct',
+			'span.geo-nondefault',
+			'#coordinates',
+		],
+	],
+	'wgExtractsExtendOpenSearchXml' => [
```
Comment (Reception123) [inline LocalSettings.php:3971]: once again, why are you setting the default only?

**Review by Reception123 — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (ugochimobi) [reply LocalSettings.php:3971]: Done

File: LocalSettings.php [hunk 2]
```diff
@@ -3956,6 +3956,22 @@
...
+		'default' => null,
```
Comment (Reception123) [inline LocalSettings.php:3972]: This isn't the extension default: https://github.com/wikimedia/mediawiki-extensions-TextExtracts/blob/master/extension.json#L61

**Review by Reception123 — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 2 above)
Comment (ugochimobi) [reply LocalSettings.php:3972]: Done

Comment (Reception123) [reply LocalSettings.php]: Please see my comment on the other PR. The default needs to match the one in extension.json AND you need to add a different value for your wiki if you need one.

**Comment by ugochimobi** [comment]
@Reception123 per this https://github.com/wikimedia/mediawiki-extensions-TextExtracts/blob/master/extension.json, this PR should be ready to be merged.

File: LocalSettings.php [hunk 3]
```diff
@@ -3956,6 +3956,36 @@
...
+			'.mw-editsection',
+			'sup.reference',
+			'ol.references',
+			'.error',
+			'.nomobile',
+			'.noprint',
+			'.noexcerpt',
+			'.sortkey',
+		],
+		'gratispaideiawiki' => [
```
Comment (Universal-Omega) [inline LocalSettings.php:3978]: This should inherit the defaults also probably (+gratispaideiawiki)

File: LocalSettings.php (hunk 1 above)
Comment (Universal-Omega) [reply LocalSettings.php:3986]: No it isn't.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (ugochimobi) [reply LocalSettings.php:3986]: How?

File: LocalSettings.php (hunk 3 above)
Comment (ugochimobi) [reply LocalSettings.php]: Do you mean, all the defaults should also be included under +gratispaideiawiki?

File: LocalSettings.php [hunk 4]
```diff
@@ -3956,6 +3956,33 @@
...
+			'sup.reference',
+			'ol.references',
+			'.error',
+			'.nomobile',
+			'.noprint',
+			'.noexcerpt',
+			'.sortkey',
+		],
+		'+gratispaideiawiki' => [
+			'.metadata', // This is a good thing to remove in any case
```
Comment (Universal-Omega) [inline LocalSettings.php:3979]: Please remove the comment.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 4 above)
Comment (Universal-Omega) [reply LocalSettings.php:3979]:
```suggestion
			'.metadata',
```

File: LocalSettings.php (hunk 3 above)
Comment (Universal-Omega) [reply LocalSettings.php]: No, I just meant you needed to add the `+` which inherits the default, and appends to it with the new array.

Comment (ugochimobi) [reply LocalSettings.php]: @Universal-Omega just as i've done?

Comment (Universal-Omega) [reply LocalSettings.php]: Yes

**Review by Universal-Omega — APPROVED** [review]
Can't confirm it works, but seems fine to me now.

**Comment by ugochimobi** [comment]
So can we ask probably @RhinosF1 to merge?

**Pull request merged**